ramls/instance-iteration/iteration-job.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Iteration job",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "Job id",
+      "type": "string",
+      "$ref": "../raml-util/schemas/uuid.schema"
+    },
+    "published": {
```

Does this refer to the number of instance ID messages published?

```suggestion
    "messagesPublished": {
```

---

Thanks, updated